### DISCUSSION TITLE
localhost:3000

### DISCUSSION THREAD

**Discussion by ixeddie** [vercel/next.js#14972] [Help]
Im about to create a nextjs app followed the instructions provided however I got a different result on the page.
![NextJS](https://user-images.githubusercontent.com/47383079/86917902-799f7e80-c158-11ea-929d-a313a76c58e7.PNG)

Supposedly this should be the page Ill be getting. 
![image](https://user-images.githubusercontent.com/47383079/86918042-ae133a80-c158-11ea-94a6-e395e93baaf7.png)

But I got this instead.
![image](https://user-images.githubusercontent.com/47383079/86918104-c71beb80-c158-11ea-8f6c-107847ec9df2.png)

Any suggestion how to correct this would be greatly appreaciated.
PS Im new to this hehe..

**Comment by jamesmosier** [comment]
Hi @ixeddie! It looks like `npm run dev` got terminated for some reason. If you try to run `npm run dev` in that directory, what is the output now?

**Reply by ixeddie** [reply]
![image](https://user-images.githubusercontent.com/47383079/86990558-6d0a3d00-c1cf-11ea-9256-7716a46ad1fd.png)
This is the output now.  :(

**Reply by jamesmosier** [reply]
sorry, what is the output in your terminal?

**Reply by ixeddie** [reply]
Thank you james  for taking the time, I was able to fix it earlier. :)
